### DISCUSSION TITLE
Request for Fine-Tuning Train Model for PP-korean_PP-OCRv4_rec_train

### DISCUSSION THREAD

**Discussion by jsjeon-xb** [PaddlePaddle/PaddleOCR#14367] [Q&A] [answered]
Dear PaddleOCR Developers,

I am currently working on fine-tuning a Korean OCR model and have been impressed by the excellent performance of the models you have provided. As far as I understand, the PP-korean_PP-OCRv3_rec_train model is available, but I couldn't find the corresponding train model for the latest PP-korean_PP-OCRv4_rec_train.

Would it be possible for you to release the training model for PP-korean_PP-OCRv4_rec_train as open source?
Thank you very much for your consideration!

**Comment by GreatV** [comment, marked as answer]
The training model for **PP-korean_PP-OCRv4_rec_train** does not appear to be officially released as of now, based on the information provided. From the related discussions and documentation, it is evident that PaddleOCR has released fine-tuning and pretraining guides for other models, such as **PP-OCRv3** and **en_PP-OCRv4_rec**, but there is no explicit mention of the Korean-specific PP-OCRv4 recognition training model being open-sourced.

### Key Points from Related Discussions:
1. **Pretrained Models for PP-OCRv4**:
   - According to [Discussion 13963](https://github.com/PaddlePaddle/PaddleOCR/discussions/13963), pretrained models like `en_PP-OCRv4_rec_train` can be downloaded and used for fine-tuning. However, there is no mention of a specific Korean training model.

2. **PP-OCRv3 vs. PP-OCRv4**:
   - In [Discussion 13964](https://github.com/PaddlePaddle/PaddleOCR/discussions/13964), it is noted that **PP-OCRv3** models are recommended for fine-tuning due to the availability of distillation models and documentation. However, for **PP-OCRv4**, there seems to be no equivalent documentation or pretraining models specifically for Korean.

3. **Fine-Tuning Guidance**:
   - The fine-tuning process for OCR models is well-documented in the [PaddleOCR Fine-Tuning Guide](https://github.com/PaddlePaddle/PaddleOCR/blob/main/doc/doc_en/finetune_en.md). While this guide applies to text recognition models like **PP-OCRv4_rec**, it requires a pretrained model to begin with. For your case, since the Korean **PP-OCRv4_rec_train** model is unavailable, you might have to either:
     - Use a general pretrained **PP-OCRv4_rec** model (like the English version) and fine-tune it with Korean-specific data.
     - Start with the Korean **PP-OCRv3_rec_train** model, which is available, as a workaround.

4. **Request for Open-Sourcing Korean PP-OCRv4_rec**:
   - Your request aligns with the concerns raised in [Discussion 13875](https://github.com/PaddlePaddle/PaddleOCR/discussions/13875), where users expressed difficulties in accessing certain pretrained models for fine-tuning. It might be helpful to directly raise an issue in the PaddleOCR repository or join the discussions to request the release of the Korean PP-OCRv4 training model.

### Recommendation:
- If you are looking to fine-tune a model for Korean OCR, you can consider using the **PP-korean_PP-OCRv3_rec_train** model and adjust the configurations for your dataset. Although it is not the latest version, it is currently the most accessible Korean-specific model.
- You can also experiment with the general **PP-OCRv4_rec** model and fine-tune it with your Korean dataset, but this might require additional effort in adapting it to Korean-specific characters and styles.

### Action Steps:
1. Monitor the [PaddleOCR Discussions](https://github.com/PaddlePaddle/PaddleOCR/discussions) and repository for updates regarding the release of the **PP-korean_PP-OCRv4_rec_train** model.
2. Consider posting a feature request directly in the PaddleOCR repository to advocate for the model's release.
3. Use the existing fine-tuning documentation as a guide to adapt other pretrained models for your use case.

If you need further assistance with fine-tuning steps or configuration, feel free to ask!

---
*Response generated by [feifei-bot](https://github.com/GreatV/feifei) | chatgpt-4o-latest*

**Reply by jsjeon-xb** [reply]
Thank you for your answer!
